### ISSUE TITLE
Populate `nps_official` field in parks table

### ISSUE THREAD

**Issue by chadlawlis** [nationalparkservice/places-data#734]
Per https://www.nps.gov/aboutus/upload/Site-Designations-04-13-16.pdf and/or http://inside.nps.gov/index.cfm?handler=parkunitlist&sort=11.

The key with the latter is to sort the list as I have, which displays the same 411 units at the top and an additional list of "Additional sites managed by or affiliated with the NPS" below (`Unit = 0`).

cc @mtaylorlong

**Comment by chadlawlis** [comment]
`nps_official = 'true'` for the 411 "official" units, `'false'` for the others.

**Comment by mtaylorlong** [comment]
I added `true/false` data for every park included in the [inside.nps.gov](http://inside.nps.gov/index.cfm?handler=parkunitlist&sort=11) list of official and unofficial units. Seven records still remain `NULL`, but we can probably assume they are either `unofficial` or too new to have been added to the list. 

There's more work to do in adding records for parks we didn't have in our database, but that's all documented in other issues. The initial population of the new `nps_official` field is done, so we can consider this issue complete.
